### ISSUE TITLE
Push notifications to all users who use my bot. 

### ISSUE THREAD

**Issue by leet896** [atipugin/telegram-bot-ruby#130]
Push notifications to all users who use 
Hello. (In advance, I'm sorry for the bad translation, I wrote through an interpreter).
Actually the question itself (I myself do not know much about bots, but I ask you to help me)
I ask you to tell me (code), I want to make a mailing (text message) to users (all) who use my bot (hence who deleted the chat with my bot, the message does not get)
You can write the code if you have such an opportunity, since I myself have not found it on the Internet.

**Comment by Defman21** [comment]
Telegram bots are not meant for that. You can create a channel, give your bot admin permissions in your channel and send something to the channel with your bot. Unless you keep user IDs somewhere, you can't find who is using your bot. If you have such a database, you can use it to send messages to the users. (`bot.send_message`).

**Comment by leet896** [comment]
I do not want to create a channel => add users etc
I need the messages to be notified through the bot to users.
You are talking about the database, I do not have it.
But as I understand it, I need to know (id of user profiles) who uses my bot.
Tell me how can I do this? It can simply make logs (for example, when the start button is pressed) and after the id of the profiles who started the bot will be saved in a text file on the server (I use the service heroku) and then I will insert the profile id into the program code from this file.
Tell me how to write this? (Make saving of id profiles in a text file and the code itself that will send a message to the user by notification)

**Comment by Defman21** [comment]
Yep. When a user activates your bot (via `/start`), you have to save the id of the **chat**, not user (`message.chat.id`) somewhere (it could be a file, a database, etc.). Then, when you want to send something to your users, you simply do e.g. `chat_ids.each { |id| bot.send_message(chat_id: id, text: "hello world") }`.
